core/templates/dev/head/domain/question/EditableQuestionBackendApiService.ts

```diff
@@ -49,7 +49,7 @@ oppia.factory('EditableQuestionBackendApiService', [
         skillId, questionDict, successCallback, errorCallback) {
       var questionCreationUrl = UrlInterpolationService.interpolateUrl(
         QUESTION_CREATION_URL, {
-          skill_ids: skillId
+          comma_separated_skill_ids: skillId
```

Comment:
So then the argument name should be commaSeparatedSkillIds right? Currently it's singular and suggests only one skill ID.

(Though really this should be part of the questionDict. I think you filed an issue for that.)

Reply:
Note: this references https://github.com/oppia/oppia/pull/6929#discussion_r294198691

Reply:
So I am a bit hesitant to change the name because in the future we will misunderstand this. As of now the frontend only supports passing of a single skill ID. The backend will accept it just fine because it can handle 1-n skill Ids. If however I change the name, it will give the impression that the frontend work of passing multiple skill IDs is done, hope this makes sense!
We are anyway moving this to questionDict as well. This entire thing won't be an issue once that is done

Reply:
Oh I think I see. In that case please add a TODO/comment here to explain all that (and maybe link to this comment thread or the issue), otherwise currently it looks pretty confusing if someone just reads it as-is.

Reply:
Done